.husky/pre-commit

```diff
@@ -1,5 +1,3 @@
-#!/usr/bin/env sh
-. "$(dirname -- "$0")/_/husky.sh"
 
```

Comment:
confirm this change.

Reply:
Most probably this was removed when upgrading the husky dependency